e2e_tests/tests/experiment/experiment.py

```diff
@@ -314,6 +314,7 @@ def check_if_string_present_in_trial_logs(trial_id: int, target_string: str) ->
     for log_line in logs:
         if target_string in log_line:
             return True
+    print(f"{target_string} not found in trial {trial_id} logs: {logs}")
```

Comment:
Can you make set `file=sys.stderr?`  I've noticed that in many run configurations, pytest will unhelpfully swallow stdout while exposing only stderr.

Reply:
done